packages/forms-web-app/__tests__/unit/controllers/before-you-start/use-existing-service-listed-building.test.js

```diff
@@ -0,0 +1,16 @@
+const useExistingServiceController = require('../../../../src/controllers/before-you-start/use-existing-service-listed-building');
```

Comment:
rename useExistingServiceListedBuildingController here and in rest of this test for clarity?